WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -386,17 +388,24 @@ public boolean isGooglePlayServicesAvailable(Activity activity) {
 
     private void initViewModel() {
         mFloatingActionButton = findViewById(R.id.fab_button);
+        mFabTooltip = findViewById(R.id.fab_tooltip);
 
         mViewModel = ViewModelProviders.of(this, mViewModelFactory).get(WPMainActivityViewModel.class);
 
         if (BuildConfig.INFORMATION_ARCHITECTURE_AVAILABLE) {
             // Setup Observers
-            mViewModel.getShowMainActionFab().observe(this, shouldShow -> {
-                if (shouldShow) {
+            mViewModel.getShowMainActionFab().observe(this, showUiState -> {
```

Comment:
maybe call it `fabUiState` instead of `showUiState`?

Reply:
Yep better naming...see here 95381cb8653731e40440a61be244ed2eeafd7664